### DISCUSSION TITLE
Error 400: Validation failed. Missing Language

### DISCUSSION THREAD

**Discussion by paultvis** [atrocore/atrocore#362] [General]
Hi,

I have been adding attributes and groups, but for some reason the forms no longer ask for the name in the one language we have configured. When I try to save a new group I get an error 'Error 400: Validation failed. 'Group Name › en_GB' is required.'. The field for this was being displayed earlier but has disappeared.

Any suggestions on how to resolve?

Thanks

Paul

**Comment by atrolex** [comment]
all languages selected?
![image](https://user-images.githubusercontent.com/33658481/173808407-28bcd02d-91c7-4589-a0f8-d7bc3a83616c.png)

**Reply by paultvis** [reply]
Where is that accessed from? We do not have the translation module installed at the moment. Maybe looking in the wrong place..

**Reply by paultvis** [reply]
We have this set 
![image](https://user-images.githubusercontent.com/31490804/173822191-3743c6c5-466c-4fb6-8024-0886150dc421.png)

**Comment by atrolex** [comment]
For every entity in the system, you have the possibility to filter displayed fields by language. It has nothing to do with translations module.
I assume your filter for en_GB is not set, that is why you don't see this field. As you don't see it you put no value into it. It causes mistake.

**Reply by atrolex** [reply]
another possible problem could be that this fields is not set to be displayed via /Administration/Layouts/....

**Reply by paultvis** [reply]
That was the issue. Field not added to the form.

Thanks
